yarn.lock

```diff
@@ -4359,6 +4361,14 @@
   resolved "https://registry.yarnpkg.com/@types/elasticsearch/-/elasticsearch-5.0.33.tgz#b0fd37dc674f498223b6d68c313bdfd71f4d812b"
   integrity sha512-n/g9pqJEpE4fyUE8VvHNGtl7E2Wv8TCroNwfgAeJKRV4ghDENahtrAo1KMsFNIejBD2gDAlEUa4CM4oEEd8p9Q==
 
+"@types/enzyme@^3.1.13":
```

Comment:
Can the existing `"@types/enzyme@^3.9.0"` package be updated to this version as well?